index.js

```diff
@@ -20,6 +20,12 @@ function increaseSpecifityOfRule(rule, opts) {
 		) {
 			return selector + opts.stackableRoot.repeat(opts.repeat);
 		}
+		
+		// Avoid prefix for :exports since :export block defines the symbols 
+		// that are going to be exported to the consumer
```

Comment:
```suggestion
		// Avoid prefix for `:export` blocks because they define the symbols 
		// that are going to be exported to the consumer,
		// https://github.com/css-modules/icss#export
```